src/routes/entry-point.ts

```diff
@@ -0,0 +1,18 @@
+import Router from '@koa/router';
+import { Context, Middleware, Next } from 'koa';
+import { schema } from 'rdf-namespaces';
+import url from 'url';
+import Routes from './index';
+
+export default (router: Router): Middleware => (
+  async ({ request, response }: Context, next: Next): Promise<void> => {
+    response.body = {
+      '@id': url.resolve(request.origin, router.url(Routes.EntryPoint, {})),
+      '@type': schema.EntryPoint,
+      [schema.name]: { '@value': 'Article Store', '@language': 'en' },
+    };
+    response.type = 'jsonld';
+
+    await next();
```

Comment:
annoying to have to remember to call `next()`; what is the use case `@koa/router` solves with this?

Reply:
Having middleware on specific routes, rather than globally. The example (https://github.com/koajs/router/blob/master/API.md#multiple-middleware) isn't particurlarly realistic. But since it's possible probably good practice to include it.

Reply:
> annoying to have to remember to call `next()`; what is the use case `@koa/router` solves with this?

On the contrary - it's good practice as it gives the the option to fail fast on the request and stop passing it along. In contrast to `python` based frameworks where using expections is the pythonic way :smile: